docs/src/modules/scheduledCallback.js

```diff
@@ -0,0 +1,29 @@
+import {
+  unstable_cancelCallback as cancelCallback,
+  unstable_scheduleCallback as scheduleCallback,
+  unstable_LowPriority as LowPriority,
+} from 'scheduler';
+
+/**
+ * implementing lodash/throttle and debounce via scheduler
+ * @param {Function} callback 
+ */
+export default function scheduledCallback(callback) {
+  let callbackNode;
+
+  function run() {
+    callbackNode = scheduleCallback(callback, LowPriority);
```

Comment:
Does the scheduler flush/cancel previously scheduled callbacks or should we cancel them manually?